src/fmu/dataio/dataio.py

```diff
@@ -769,15 +766,10 @@ def generate_metadata(
 
         self._update_fmt_flag()
         fmudata = self._get_fmu_provider() if self._fmurun else None
-        self._metadata = drop_nones(
-            generate_export_metadata(
-                obj, self, fmudata, compute_md5=compute_md5
-            ).model_dump(mode="json", exclude_none=True, by_alias=True)
-        )
```

Comment:
I think `drop_nones` can be dropped across the board with `exclude_none=True`?

From pydantic docs:

> exclude_none 	[bool](https://docs.python.org/3/library/functions.html#bool) 	Whether to exclude fields that have a value of None.

Reply:
I have been wondering the same, and I think it is.. but did not dare to make the change 😄

Reply:
As far as I can tell this is OK to do, and we should do it: #693 693